client/homepage/stats-overview/style.scss

```diff
@@ -85,7 +85,7 @@ article.woocommerce-stats-overview__install-jetpack-promo {
 			}
 		}
 
-		.woocommerce-layout & button.components-button.is-button.is-primary {
+		.woocommerce-layout & button.components-button.is-primary {
 			margin-left: 24px;
 			background: transparent;
```

Comment:
Is this supposed to be transparent?  If so, I think this button should use the `isSecondary` prop now.  Otherwise we can remove this background property.

<img width="391" alt="Screen Shot 2020-06-12 at 11 47 46 AM" src="https://user-images.githubusercontent.com/10561050/84484133-0151b480-aca3-11ea-878a-142fd7dad173.png">